vcd/provider.go

```diff
@@ -111,6 +111,7 @@ func Provider() terraform.ResourceProvider {
 			"vcd_lb_virtual_server":  resourceVcdLBVirtualServer(),  // 2.4
 			"vcd_nsxv_dnat":          resourceVcdNsxvDnat(),         // 2.5
 			"vcd_nsxv_snat":          resourceVcdNsxvSnat(),         // 2.5
+			"vcd_nsxv_firewall":      resourceVcdNsxvFirewall(),     // 2.5
```

Comment:
Important. I strongly suggest renaming this resource and data source from `vcd_nsxv_firewall` -> `vcd_nsxv_firewall_rule`. Reasons:
- In order to create more than one rule, you need to create more than one resources.
- When you create more than one, current `vcd_nsxv_firewall` name loses rational, because it looks as if you're defining multiple firewalls (as in multiple gateways).
- It's just more intuitive to see "rule" in the resource name.

CC: @vbauzysvmware @dataclouder

Reply:
I agree, I mentioned similar thing in my review.